hat/assets/js/apps/Iaso/domains/mappings/components/Filter.tsx

```diff
@@ -0,0 +1,145 @@
+import React, { useState, FunctionComponent, useCallback } from 'react';
+
+import { Grid, Button, Box, useMediaQuery, useTheme } from '@mui/material';
+import { makeStyles } from '@mui/styles';
+import SearchIcon from '@mui/icons-material/Search';
+import { commonStyles, useSafeIntl } from 'bluesquare-components';
+
+import InputComponent from '../../../components/forms/InputComponent';
+import { useFilterState } from '../../../hooks/useFilterState';
+import { mappingTypeOptions } from './MappingTypeOptions';
+
+import MESSAGES from '../messages';
+
+import { baseUrl } from '../config';
+import { useGetFormsDropdownOptions } from '../hooks/requests/useGetFormsDropdownOptions';
+import { useGetProjectsDropdownOptions } from '../../projects/hooks/requests';
+import { useGetOrgUnitTypes } from '../../orgUnits/hooks/requests/useGetOrgUnitTypes';
+
+const useStyles = makeStyles(theme => ({
+    ...commonStyles(theme),
```

Comment:
You are only using styles for the Button, are you sure this is required ? You can probably avoid this by using sx prop or a wrap the button into a box